src/Backoffice/Employee/App/Requests/EmployeeRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace src\Backoffice\Employee\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class EmployeeRequest extends FormRequest
```

Comment:
```suggestion
class StoreEmployeeRequest extends FormRequest
```
Always use the same naming for Requests, Controllers and Actions